system-test/spanner.ts

```diff
@@ -4300,9 +4456,10 @@ describe('Spanner', () => {
       insertThenUpdateRow(done, postgreSqlTable);
     });
 
-    describe('insert & query', () => {
+    describe.skip('insert & query', () => {
```

Comment:
Do not skip this, this is also used for other types. Instead comment the part where we are adding Float32 and add a comment for later

Reply:
Done